docs/sources/operations/storage/retention.md

```diff
@@ -5,33 +5,31 @@ title: Retention
 
 Retention in Loki is achieved either through the [Table Manager](#table-manager) or the [Compactor](#Compactor).
 
-## Which one should you use ?
+Which one should you use ?
```

Comment:
Remove this line completely.  We don't need the question, since the sentence above it implies that we'll be describing both.